xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -20,8 +20,40 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents an assembly load context, which represents the runtime's concept of a load binding scope. This class is abstract and must be implemented to be instantiated.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> represents an instance of a load context.  Conceptually, a load context creates a scope for loading, resolving, and potentially unloading a set of assemblies.
+ 
+ <xref:System.Runtime.Loader.AssemblyLoadContext> is an abstract class. While the class is abstract, only the <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> needs to be implemented to create a concrete class.
+
+### Usage in the runtime
+ The runtime implements two <xref:System.Runtime.Loader.AssemblyLoadContext>:
+     * The <xref:System.Runtime.Loader.AssemblyLoadContext.Default?displayProperty=nameWithType> represents the runtimes default binding context which is used for the framework and the application main assembly and its normal dependencies.
+     * The <xref:System.Reflection.Assembly.LoadFile(System.String)?displayProperty=nameWithType> method isolates the assemblies it loads by instantiating the most basic <xref:System.Runtime.Loader.AssemblyLoadContext>.  It has a simplistic isolation scheme which loads each assembly in its own <xref:System.Runtime.Loader.AssemblyLoadContext> with no dependency resolution.
+
+### Application usage
+ An application can create its own <xref:System.Runtime.Loader.AssemblyLoadContext> to create a custom solution for advanced binding scenarios. The customization focuses on defining dependency resolution mechanisms.
```

Comment:
Isolation is also a strong motivation to use ALC, and we should mention that here.

Reply:
@swaroop-sridhar this and other suggestions done.  Thanks for the solid feedback